src/main/java/kata/makers/ShowsDatabase.java

```diff
@@ -0,0 +1,38 @@
+package kata.makers;
+
+import java.util.ArrayList;
+import java.util.List;
+import kata.makers.exception.RequiredFieldsEmptyException;
+import lombok.Getter;
+
+@Getter
+public class ShowsDatabase {
+
+  private final List<Show> shows = new ArrayList<>();
```

Comment:
A Java List allows for duplicates and null values. Additionally, a list can only contain items you can check for their properties, but don't have keys to search by.

Is your plan to account for this in code or are you planning on refactoring to list types that may be easier to navigate and control?